app/routers/api/browser.py

```
@@ -1,4 +1,4 @@
-# ruff: noqa: B008
+# ruff: noqa: B008, PLR0912, PLR0915, TRY301
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLR0912`, `PLR0915`)

Remove unused `noqa` directive

(RUF100)